docker-compose.yml

```diff
@@ -1,7 +1,9 @@
 version: '2'
 services:
   soulseek:
-    build: .
+    container_name: soulseek
```

Comment:
Would it not be better to leave out the container name, allowing for more than one stacks to be initiated? Otherwise it would complain the container name is already used.

Reply:
It's what major entertainment services do, ex: https://github.com/linuxserver/docker-plex#docker-compose-recommended
Because those who have multiple instances are the minority, not the norm. So you're making life easier for a majority.